lib/web/assistant.go

```diff
@@ -275,6 +283,9 @@ func runAssistant(h *Handler, w http.ResponseWriter, r *http.Request, sctx *Sess
 		// write user message to both in-memory chain and persistent storage
 		chat.Insert(openai.ChatMessageRoleUser, wsIncoming.Payload)
 
+		// TODO(justinas): should we handle this error?
```

Comment:
Print it in logs (not as an error) and make sure that `promptTokens` is 0. I don't think that having (if this ever happens) a 99% accurate token usage is an issue.